src/Microsoft.Crank.Agent/Startup.cs

```diff
@@ -1681,7 +1681,9 @@ private static async Task<(string containerId, string imageName, string workingD
 
             var command = OperatingSystem == OperatingSystem.Linux
                 ? $"run -d {environmentArguments} {job.Arguments} --label benchmarks --name {containerName} --privileged --network host {imageName} {source.DockerCommand}"
-                : $"run -d {environmentArguments} {job.Arguments} --label benchmarks --name {containerName} --network SELF --ip {hostname} {imageName} {source.DockerCommand}";
+                : string.Equals(hostname, "localhost", StringComparison.OrdinalIgnoreCase)
+                    ? $"run -d {environmentArguments} {job.Arguments} --label benchmarks --name {containerName} -p {job.Port}:{job.Port} {imageName} {source.DockerCommand}"
+                    : $"run -d {environmentArguments} {job.Arguments} --label benchmarks --name {containerName} --network SELF --ip {hostname} {imageName} {source.DockerCommand}";
```

Comment:
I had to run this to be able to run locally on windows without having to create a network and to be able to map the port.
Not sure however if that `Port` parameter is the correct parameter.

Maybe there should be a more generic argument to pass parameter to docker while I believe currently you can only specify arguemnts to the container

Reply:
I think we can just keep the port:port version on Windows. I used SELF because I was trying to create a faster network interface, but at least if port:port works it's better for now.